pom.xml

```diff
@@ -77,8 +77,9 @@ flexible messaging model and an intuitive client API.</description>
   </issueManagement>
 
   <properties>
-    <maven.compiler.source>8</maven.compiler.source>
-    <maven.compiler.target>8</maven.compiler.target>
+    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>17</maven.compiler.target>
+    <pulsar.client.javac.release>8</pulsar.client.javac.release>
```

Comment:
I think it would be more consistent to use `compiler` instead of `javac`, thus renaming `pulsar.client.javac.release` -> `pulsar.client.compiler.release`.

Reply:
updated.